company/team/index.md

```diff
@@ -2,7 +2,12 @@
 
 This page contains brief bios of our team. Teammates may also have a personal documentation page in this directory that is named according to their Sourcegraph email address (e.g. you@sourcegraph.com -> you.md).
 
-To add yourself to this page, copy the following template, paste it at the end of this file, and make it about yourself!
+To add yourself to this page:
+1. Accept GitHub's email invite to the Sourcegraph org (and then make sure that you are added to the appropriate teams).
```

Comment:
```suggestion
1. Accept GitHub's email invite to the Sourcegraph org (you should be a member of [the `Everyone` group](https://github.com/orgs/sourcegraph/teams/everyone) in Sourcegraph's GitHub organization).
```

Reply:
Updating this to make it more clear how to debug the potential problem here.

Reply:
And let me know if you are familiar with how to accept my suggestion or if I can help with that